vscode-wpilib/resources/gradle/shared/.gitignore

```diff
@@ -75,6 +75,84 @@ hs_err_pid*
 # .nfs files are created when an open file is removed but is still being accessed
 .nfs*
 
+### Python ###
+
+# Byte-compiled / optimized / DLL files
+__pycache__/
+*.py[cod]
+*$py.class
+
+# C extensions
+*.so
```

Comment:
This one scares me. I've seen people put these in the deploy folder. We should instead be blocking folders, not files.